umm-spec-lib/test/cmr/umm_spec/test/umm_g/granule.clj

```diff
@@ -3,152 +3,34 @@
-                                     ["QAFRACTIONGOODQUALITY Value7"
-                                      "QAFRACTIONGOODQUALITY Value8"]})
-                                  (umm-lib-g/map->ProductSpecificAttributeRef
-                                    {:name "QAFRACTIONNOTPRODUCEDCLOUD"
-                                     :values
-                                     ["QAFRACTIONNOTPRODUCEDCLOUD Value9"
-                                      "QAFRACTIONNOTPRODUCEDCLOUD Value10"]})]
-    :spatial-coverage (umm-lib-g/map->SpatialCoverage
-                       {:geometries
-                        [(p/point -77 88) (p/point 10 10)
-                         (mbr/mbr -180 85.04450225830078 180 -85.04450225830078)
-                         (poly/polygon
-                          [(umm-s/ring
-                            [(p/point -10 -10) (p/point 10 -10) (p/point 10 10) (p/point -10 10) (p/point -10 -10)])
-                           (umm-s/ring
-                            [(p/point -5 -5) (p/point -1 -5) (p/point -1 -1) (p/point -5 -1) (p/point -5 -5)])
-                           (umm-s/ring
-                            [(p/point 0 0) (p/point 5 0) (p/point 5 5) (p/point 0 5) (p/point 0 0)])])
-                         (l/line-string [(p/point -100 -70) (p/point -88 -66)])]})
-    :related-urls [(umm-c/map->RelatedURL
-                    {:type "GET DATA"
-                     :url "https://daac.ornl.gov/daacdata/islscp_ii/vegetation/erbe_albedo_monthly_xdeg/data/erbe_albedo_1deg_1986.zip"
-                     :description "This link provides direct download access to the granule."
-                     :mime-type "application/zip"
-                     :title "This link provides direct download access to the granule."
-                     :size 395.673})
-                   (umm-c/map->RelatedURL
-                    {:type "VIEW RELATED INFORMATION"
-                     :sub-type "USER'S GUIDE"
-                     :url "https://daac.ornl.gov/ISLSCP_II/guides/erbe_albedo_monthly_xdeg.html"
-                     :description "ORNL DAAC Data Set Documentation"
-                     :mime-type "text/html"
-                     :title "ORNL DAAC Data Set Documentation"})
-                   (umm-c/map->RelatedURL
-                    {:type "GET RELATED VISUALIZATION"
-                     :url "https://webmap.ornl.gov/sdat/pimg/957_1.png"
-                     :description "ISLSCP II EARTH RADIATION BUDGET EXPERIMENT (ERBE) MONTHLY ALBEDO, 1986-1990"
-                     :mime-type "image/png"
-                     :title "ISLSCP II EARTH RADIATION BUDGET EXPERIMENT (ERBE) MONTHLY ALBEDO, 1986-1990"
-                     :size 10})]}))
-
 (deftest parse-granule-test
   (testing "parse granule"
-    (is (= expected-granule (core/parse-metadata {} :granule :umm-json sample-umm-g-granule)))))
+    (is (= expected-util/expected-sample-granule (core/parse-metadata {} :granule :umm-json sample-umm-g-granule)))))
```

Comment:
It feels wrong to have the `sample-umm-g-granule` in one namespace and the expected-sample-granule` in a different namespace away from the test.

Reply:
Yea, its how we do it in umm-c generate_and_parse so I figured it was ok here.